viewer/packages/tools/src/HtmlOverlay/HtmlOverlayTool.test.ts

```diff
@@ -78,15 +78,16 @@ describe(HtmlOverlayTool.name, () => {
     expect(htmlElement.style.top).toBeEmpty();
     expect(htmlElement.style.left).toBeEmpty();
     const position = new THREE.Vector3(0, 0, 0.5);
+    const renderSize = renderer.getSize(new THREE.Vector2());
 
     // Act
     helper.add(htmlElement, position);
     helper.forceUpdate();
 
     // Assert
     expect(htmlElement.style.visibility).toBe('visible');
-    expect(htmlElement.style.top).toBe(`${renderer.domElement.height / 2}px`);
-    expect(htmlElement.style.left).toBe(`${renderer.domElement.width / 2}px`);
+    expect(htmlElement.style.left).toBe(`${renderSize.x / 2}px`);
+    expect(htmlElement.style.top).toBe(`${renderSize.y / 2}px`);
```

Comment:
Why change the test? Why is center of dom element not valid? I'm a bit scared that we change the verification here